contracts/eosio.system/eosio.system.hpp

```diff
@@ -19,7 +20,7 @@ namespace eosiosystem {
    /**
    * We create the native EOSIO token type
    */
-   typedef eosio::token<uint64_t,N(eosio)> native_tokens;
+   typedef eosio::token<N(eosio), S(4,EOS), uint64_t> native_tokens;
```

Comment:
What's the meaning of `S(4,EOS)`, where is `S()` defined?

Reply:
Answer to myself: 
`S()` macro is defined in`#define S(P,X) ::eosio::string_to_symbol(P,#X)` 
- eosio.system.hpp has `#included <token.hpp>`, 
- token.hpp has `#include<assets.hpp>`